src/couch_views/src/couch_views_indexer.erl

```diff
@@ -541,3 +559,35 @@ key_size_limit() ->
 
 value_size_limit() ->
     config:get_integer("couch_views", "value_size_limit", ?VALUE_SIZE_LIMIT).
+
+
+active_tasks_info(ChangesDone, DbName, DDocId, LastSeq, DBSeq) ->
+    VS = case LastSeq of
```

Comment:
I don't like storing all this active task info in the job. All of this can be generated when CouchDB receive's a `_active_tasks` request. If we keep adding more and more information to the job info means we running the risk of exceeding the FDB value limit. Rather just add the `current_version_stamp` and `db_version_stamp` to the job options. Is `indexer_pid` actually useful? What would a user do with that?

Reply:
The reason I had thought of it is that for replication tasks, for example, we we'd have to know to introspect a few other places to know how to add "source", "target", etc. So active tasks would have to start knowing about how to extract fields from various applications. Perhaps we'd want a simple registration like we have for indices in fabric2_index: https://github.com/apache/couchdb/blob/prototype/fdb-layer/src/fabric/src/fabric2_index.erl#L51-L53 - modules (applications) register a callback for _active_tasks handler and call it when building the response? Or, perhaps we can do that as round 2 if we see the values getting close to 100KB size.

Reply:
@garrensmith @nickva how about leaving just `changes_done` to the job_data and then computing the rest on the fly? It seems we should be adding an entirely new fabric_active_tasks module which allows the extractions to reside int here

Reply:
That sounds good.